readme.md

```diff
@@ -89,3 +89,29 @@ Complete the following exercises to practice using SQL.
 * List all books. Display the book_id, title, and a count of how many editions each book has. (hint: requires GROUP BY and JOIN)
 
 ####YAY! You're done!!
+
+1.) `SELECT * FROM subjects ORDER BY subject ASC;`
+2.) `SELECT * FROM subjects ORDER BY location ASC;`
+
+1.) `select * from books WHERE title ILIKE 'Little Women';`
+2.) `select * from books WHERE title ILIKE '%python%'; `
+3.) `SELECT * FROM subjects WHERE location ILIKE 'Main St' ORDER BY subject ASC;`
+
```

Comment:
Good, an alternative to using ilike is - select * from books where title = 'Little Women';